examples/nas/hub/searchspace/autoformer.py

```diff
@@ -0,0 +1,219 @@
+import torch
```

Comment:
I'll test this when the features it relies on are merged, including #4609, #4598, #4524.

If you are in a hurry to merge this in and close this work item, I can talk to @QuanluZhang about merging it first and doing the tests later. Otherwise, let's hold on a minute.